tests/e2e/e2e.go

```diff
@@ -59,6 +59,10 @@ const (
 	// checks. Useful for speeding up iteration during test
 	// development.
 	SkipBootstrapChecksEnvName = "E2E_SKIP_BOOTSTRAP_CHECKS"
+
+	// Start time must be a minimum of 15s ahead of the current time
+	// or validator addition will fail.
+	DefaultValidatorStartTimeDiff = 20 * time.Second
```

Comment:
nit: comment and value are different (15 and 20 seconds).
also nit: I'd rewrite the min start time as  `txexecutor.SyncBond.Add(5*time.Second)` i.e. as the require minimum star time to let a tx be accepted in mempool (see [here](https://github.com/ava-labs/avalanchego/blob/dev/vms/platformvm/block/builder/builder.go#L250)) plus a buffer